restartmanager/restartmanager.go

```diff
@@ -95,7 +95,6 @@ func (rm *restartManager) ShouldRestart(exitCode uint32, hasBeenManuallyStopped
 	go func() {
 		select {
 		case <-rm.cancel:
-			ch <- fmt.Errorf("restartmanager canceled")
```

Comment:
This is used to notify that cancel happened on the wait period(restarting state) and the state needs to be set as exited. https://github.com/docker/docker/blob/a4030787f537e79242888cc3fd8b5f3f51ce357f/libcontainerd/container_linux.go#L131